Integrations/integration-IntSights.yml

```diff
@@ -832,10 +844,76 @@ script:
         })
 
 
+    def get_mssp_sub_accounts():
+        account_id = demisto.getParam('credentials')['identifier']
+        accounts = req('GET', 'public/v1/mssp/customers').json()
+        demisto.results({
+            'Type': entryTypes['note'],
+            'EntryContext': {'IntSights.MsspAccounts': accounts},
```

Comment:
Should Be `IntSight.MsspAccount`, also need to use DT for unique entries (e.g., `IntSight.MsspAccount(val.ID === obj.ID)`)

Reply:
@liorblob, where can I read more about this notation? I don't really understand the concept.